development-updates.md

```diff
@@ -94,6 +95,10 @@ Development updates from [`edgee`](https://github.com/EagleEdge)
 
 [Update #2](https://hackmd.io/@EagleEdge/ryMs_ul4s)
 
+### edgee: Update 3
+
+[Update #3](https://hackmd.io/@EagleEdge/H1_THfwNi)
```

Comment:
Hi @EagleEdge I am getting 403 while trying to get to this page.

Reply:
Can you try it again? Apparently I forgot to press 'publish" button.

Reply:
All good now, page is available